### PR TITLE
Show clicked image in large view

### LINKED ISSUE #4290: Issues with imagery time strip view
Testathon 9-28-21, NOT fixed: clicking an image in the thumbstrip view should display the same image paused in large view, but instead always displays the latest image.
Also, if the time strip view is using it's own independent time context, use that in the preview.

image

image

Fixed in : https://github.com/nasa/openmct/pull/4280

_Originally posted by @charlesh88 in https://github.com/nasa/openmct/issues/2977#issuecomment-929664652_

Also, Imagery time strip sometimes shows placeholders in front of the marker lines.
image

Fixed in: https://github.com/nasa/openmct/pull/4280
_Originally posted by @jvigliotta  in https://github.com/nasa/openmct/issues/2977#issuecomment-929674669

### PR DESCRIPTION
**Pull request by shefalijoshi**
Resolves #4290

@charlesh88 We've switched from using SVG elements to HTML elements. Please take a look at the CSS and refine as you see fit.

### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this change backwards compatible? Will users need to change how they are calling the API, or how they've extended core plugins such as Tables or Plots?

### Author Checklist

* [ ] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [ ] Command line build passes?
* [ ] Has this been smoke tested?
* [ ] Testing instructions included in associated issue?

### CODE REVIEW

**Comment by akhenry** [comment]
@shefalijoshi  Is this pointing to the right issue? The linked issue is closed and verified?

File: src/plugins/imagery/components/ImageryView.vue [hunk 1]
```diff
@@ -653,7 +707,7 @@ export default {
             window.clearInterval(this.durationTracker);
         },
         updateDuration() {
-            let currentTime = this.openmct.time.clock() && this.openmct.time.clock().currentValue();
+            let currentTime = this.timeContext.clock() && this.timeContext.clock().currentValue();
             this.numericDuration = currentTime - this.parsedSelectedTime;
```
Comment (nikhilmandlik) [inline src/plugins/imagery/components/ImageryView.vue:711]: when `this.timeContext.clock()` is null then `currentTime` ll be `undefined` and ll cause `this.numericDuration` to be `NaN`

**Review by nikhilmandlik — CHANGES_REQUESTED** [review]
Check edge case for `this.numericDuration` else looks good

File: src/plugins/imagery/components/ImageryView.vue (hunk 1 above)
Comment (shefalijoshi) [reply src/plugins/imagery/components/ImageryView.vue]: Ok I've updated this.

**Review by nikhilmandlik — APPROVED** [review]
LGTM

**Comment by shefalijoshi** [comment]
> @shefalijoshi Is this pointing to the right issue? The linked issue is closed and verified?

That's odd. I've created a new issue for these issues: #4290

**Comment by nikhilmandlik** [comment]
### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)

**Pull request merged**